Core/include/Acts/MagneticField/NullBField.hpp

```diff
@@ -25,14 +25,6 @@ class NullBField final : public MagneticFieldProvider {
   /// @brief Default constructor
   NullBField() = default;
 
-  /// @copydoc MagneticFieldProvider::getField(const Vector3&)
-  ///
-  /// @note The @p position is ignored and only kept as argument to provide
-  ///       a consistent interface with other magnetic field services.
-  Vector3 getField(const Vector3& /*position*/) const override {
-    return m_BField;
-  }
-
```

Comment:
Similarly to what you have done with the Constant B field you could have added something like : `Vector3 getField() const { return m_BField; }` but I don't have a strong opinion on that.

Reply:
This overload here wasn't used anywhere, and only added to comply with `MagneticFieldProvider` so I thought it's best to remove this.